docs/BuildFromSource.md

```diff
@@ -33,6 +33,7 @@ Building ASP.NET Core on Windows requires:
     ```
 
     However, the build should find any JDK 11 or newer installation on the machine.
+   * Set JAVA_HOME environment variable with the path of java installation directory (Gradle needs this for execution)
```

Comment:
```suggestion
   * Set the `JAVA_HOME` environment variable with the path of java installation directory if your installation did not do that automatically and you didn't use the InstallJdk.ps1 script. (Gradle needs this for execution.)
```

Reply:
I think even with InstallJdk.ps1, JAVA_HOME would need to be set
```suggestion
   * Set the `JAVA_HOME` environment variable with the path of java installation directory if your installation did not do that automatically. (Gradle needs this for execution.)
```